README.md

```diff
@@ -1,2 +1,79 @@
-# simple-docker-harness
-Docker Harness to facilitate interaction with docker with Docker Machine and CircleCI (public)
+# Docker Harness
+
+Facilitate building and releasing docker containers
+
+## General
+
+Run `make help` to get a list of available build targets
+
+```
+Available targets:
+
+  deps                      Configure all dependencies
+  help                      This help screen
+  machine:deps              Setup dependencies for docker-machine
+  machine:info              Docker Machine Info
+  machine:recreate          Recreate the docker machine
+  machine:create            Create docker machine for environment
+  machine:destroy           Destroy docker machine for environment
+  machine:start             Start docker machine for environment
+  machine:stop              Stop docker machine for environment
+  machine:restart           Restart docker machine for environment
+  docker:build              Build a docker image
+  docker:push               Push image to Docker Hub
+  docker:pull               Pull docker image from Docker Hub
+  docker:tag                Tag the last built image with `DOCKER_TAG`
+  docker:clean              Remove existing docker images
+  docker:run                Test drive the image
+  docker:shell              Run the container and start a shell
+  docker:attach             Attach to the running container
+  docker:login              Login to docker registry
+  docker:export             Export docker images to file
+  docker:import             Import docker images from file
+```
+
+## Using Docker Harness in other repositories
+
+1. Create a `Makefile` in in the project (if one does not already exist). 
+1. Add the following snippet near the top of the project's `Makefile`
+
+*Snippet*:
+```
+export DOCKER_HARNESS_PATH ?= $(shell until [ -d docker-harness ]; do cd ..; done; pwd)/docker-harness/
```

Comment:
Its also easy enough for people to set `DOCKER_HARNESS_PATH` in their shell which will be used